tests/addresses/e2e/features/receive-money.feature

```diff
@@ -31,6 +31,19 @@ Feature: Receive money
     When I click the ShowUsed switch
     Then I should see 1 addresses
 
+  @byron
+  Scenario: Check if wallet address belong to active wallet
+    Given I am on the "Target Wallet" wallet "receive" screen
+    And I enter wallet password in generate address input field "Secret1234"
+    And I create 1 addresses
+    Then I should see the following addresses:
+      | ClassName          |
+      | generatedAddress-1 |
+      | generatedAddress-2 |
+    Then The active address should be the newest one
+    And I am on the "Test Wallet" wallet "receive" screen
+    Then The active address belongs to "Test Wallet" wallet
```

Comment:
@tomothespian please add few more steps after this one:
- Change active wallet
- Check if active address belongs to that one

Reply:
@nikolaglumac I'm not sure that I understand 😕 . When the test starts we are on **"Target Wallet"** and we generate an address and after that, we check if the newly generated address is the active address for that wallet (Target Wallet). After that, we are on **"Test Wallet"** and we are checking if the active address belongs to that wallet (new active wallet - "Test Wallet") so that address is not last generated (generated for "Target Wallet")

Reply:
I was wrong. The test works as expected 👍